graphql/graphql.go

```diff
@@ -1158,7 +1158,7 @@ func NewHandler(be *eth.EthAPIBackend) (http.Handler, error) {
 	h := &relay.Handler{Schema: s}
 
 	mux := http.NewServeMux()
-	mux.Handle("/", GraphiQL{})
+	mux.HandleFunc("/", (GraphiQL{}).webHandler)
```

Comment:
Any particular reason for this? I think the old one was better because you can use GraphiQL from the outside too. This way it become unusable as a library.

Reply:
Ah.. It is the remainder of an unsuccessful experiment. I'm changing it